.github/workflows/test-browser.yml

```diff
@@ -73,7 +73,7 @@ jobs:
       - name: Node.js setup
         uses: actions/setup-node@v2
         with:
-          node-version: ${{ matrix.node }}
```

Comment:
`matrix.node` is not defined in this workflow, it was defaulting to NodeJS 16. So this change is effectively a noop.